internal/services/notification/action/action.go

```diff
@@ -0,0 +1,36 @@
+package action
```

Comment:
Probably actionHandler shouldn't belong to this commit but to the commit that add the apis handlers.